packages/envconfig/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@temporalio/envconfig",
+  "version": "1.0.0",
```

Comment:
Let's set the initial version to `0.0.1` to avoid confusion with something that would have actually existed in SDK 1.0.0. It will automatically get bumped to `1.13.2` (or whatever, same as other packages) when we publish a first release that includes it.